From d773e8f744bccd02abcb92e20bf21dfb2ead9701 Mon Sep 17 00:00:00 2001
From: Pierre-Yves <pyu@riseup.net>
Date: Mon, 23 Jul 2018 11:55:05 +0200
Subject: [PATCH 2/2] Cleanup Makefile

Do not install useless docs (COPYING, INSTALL, NEWS, ..)
Do not create runstatedir

Signed-off-by: Pierre-Yves <pyu@riseup.net>
---
 Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 3f6758b..38a121b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,7 +8,6 @@ dist_stubbyconf_DATA = stubby.yml.example
 
 dist_noinst_DATA = doc/stubby.1.in
 dist_man_MANS = stubby.1
-dist_doc_DATA = AUTHORS COPYING ChangeLog INSTALL NEWS README.md
 
 %.1 :: doc/%.1.in ; sed -e "s|@ETCDIR@|$(stubbyconfdir)|g" $< > $@
 
@@ -25,4 +24,3 @@ stubby.yml.windows: ${srcdir}/stubby.yml.example
 install-data-hook: $(CONFIG_FILE)
 	test -f $(DESTDIR)${stubbyconfdir}/stubby.yml || \
 		${INSTALL_DATA} $(CONFIG_FILE) $(DESTDIR)${stubbyconfdir}/stubby.yml
-	$(INSTALL) -m 755 -d $(DESTDIR)$(runstatedir)
-- 
2.18.0

